efi/fw_load_handler.go

```diff
@@ -101,32 +99,32 @@ func (h *fwLoadHandler) measureSecureBootPolicyPreOS(ctx pcrBranchContext) error
 		events = events[1:]
 
 		switch {
-		case e.PCRIndex < secureBootPCR && e.EventType == tcglog.EventTypeSeparator:
+		case e.PCRIndex < tcglog.PCRIndex(secureBootPolicyPCR) && e.EventType == tcglog.EventTypeSeparator:
```

Comment:
would it make sense to change the type in Event?

Reply:
Possibly. I think at one point I tried to implement the tcglog-parser library without a dependency on go-tpm2, with it's own types and constants for digest algorithms as well, but it's had a dependency on it for a while now so it doesn't make much sense to keep the `tcglog.PCRIndex` type. I'll fix that up separately.